src/ghga_service_commons/api/api.py

```diff
@@ -217,6 +219,21 @@ async def correlation_id_middleware(
         return response
 
 
+async def request_logging_middleware(request: Request, call_next):
+    url = request.url
+    start_time = time.time()
+    response = await call_next(request)
+    duration = round((time.time() - start_time) * 1000)
+    try:
+        status_phrase = http.HTTPStatus(response.status_code).phrase
+    except ValueError:
+        status_phrase = ""
+    log.info(
+        f'{request.method} {url} "{response.status_code} {status_phrase}" - {duration}ms'
```

Comment:
```suggestion
        f'{request.method} {url} "{response.status_code} {status_phrase}" - {duration} ms'
```
```suggestion
        f'{request.method} {url} "{response.status_code} {status_phrase}" - {duration} ms'
```

Reply:
Would it make sense to pass the parts of the message - particularly the duration - in the "details" kwarg?